std_msgs/Header header
geometry_msgs/Point32[] vertices
geometry_msgs/Vector3 normal
float32[4] plane_eq
float32 area
